components/NavigationDrawer/src/MDCBottomDrawerPresentationController.m

```diff
@@ -0,0 +1,146 @@
+/*
+ Copyright 2018-present the Material Components for iOS authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import "MDCBottomDrawerPresentationController.h"
+
+#import "MDCBottomDrawerViewController.h"
+#import "private/MDCBottomDrawerContainerViewController.h"
+
+static UIColor *MDCBottomDrawerOverlayBackgroundColor(void) {
+  return [UIColor colorWithWhite:0 alpha:0.4f];
```

Comment:
Can we get these colors from a color scheme?  For instance, in a darker theme would the background color be a translucent white color?

Reply:
adding a color themer support to this component will be a follow up

Reply:
A link to a newly opened tracking issue can be useful if the followup isn't going to be immediate.

Reply:
https://github.com/material-components/material-components-ios/issues/4910